meson.build

```
@@ -22,15 +23,19 @@ import('pkgconfig').generate(
 	name: meson.project_name(),
 	description: 'JIT assembler for x86(IA32), x64(AMD64, x86-64)',
 	version: meson.project_version(),
-	url: 'https://github.com/herumi/xbyak'
+	url: 'https://github.com/herumi/xbyak',
+	variables: ['includedir=@0@'.format(include_dir)],
+	dataonly: true,
 )
 
+shared_cmake_dir = get_option('prefix') / 'share/cmake/xbyak'
```

It isn't necessary to have an absolute path here, so `get_option('prefix') /` can be dropped. Also, `share` shouldn't be hardcoded, but `datadir` should be used instead.

```suggestion
shared_cmake_dir = get_option('datadir')/'cmake'/meson.project_name()
```

Also, as a minor nitpick, I'd personally move the variable into the `if` block below.